### PR TITLE
[FIX] Test & Learn: number of folds causes many errors

### PR DESCRIPTION
**Pull request by jerneju**
##### Issue
https://sentry.io/biolab/orange3/issues/267670404/
https://sentry.io/biolab/orange3/issues/269582517/
https://sentry.io/biolab/orange3/issues/282342454/
https://sentry.io/biolab/orange3/issues/282342206/
https://sentry.io/biolab/orange3/issues/196744688/

##### Description of changes
ContinuousVariable with Folds now contains string values instead of int.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by jerneju** [comment]
https://sentry.io/biolab/orange3/issues/267670404/

**Comment by jerneju** [comment]
https://sentry.io/biolab/orange3/issues/269582517/

**Comment by jerneju** [comment]
https://sentry.io/biolab/orange3/issues/282342454/

**Comment by jerneju** [comment]
https://sentry.io/biolab/orange3/issues/282342206/

**Comment by jerneju** [comment]
https://sentry.io/biolab/orange3/issues/196744688/

File: Orange/evaluation/testing.py [hunk 1]
```diff
@@ -276,7 +276,7 @@ def get_augmented_data(self, model_names, include_attrs=True, include_prediction
 
         # add fold info
         if self.folds is not None:
-            new_meta_attr.append(DiscreteVariable(name="Fold", values=[i+1 for i, s in enumerate(self.folds)]))
+            new_meta_attr.append(DiscreteVariable(name="Fold", values=[str(i+1) for i, s in enumerate(self.folds)]))
```
Comment (kernc) [inline Orange/evaluation/testing.py]:
```py
# alternatives
[str(i) for i, _ in enumerate(self.folds, 1)]
[str(i) for i in range(1, len(self.folds) + 1)]
np.arange(1, len(self.folds) + 1, dtype=str).tolist()
```

Comment (jerneju) [inline Orange/evaluation/testing.py]: @kernc 
`# the second alternative below is the fastest:
[i+1 for i, s in enumerate(self.folds)]  # 0.957
[i+1 for i, _ in enumerate(self.folds)]  # 0.916
[str(i) for i, _ in enumerate(self.folds, 1)]  # 2.527
[str(i) for i in range(1, len(self.folds) + 1)]  # 2.417
np.arange(1, len(self.folds) + 1, dtype=str).tolist()  # does not work`

**Pull request merged**